srcpkgs/ghidra/template

```diff
@@ -0,0 +1,32 @@
+# Template file for 'ghidra'
+pkgname=ghidra
+version=9.1
+revision=1
+_gitver=f8994f448826e1946af958980b8481c68b5c5ff2
```

Comment:
Why do you use a git commit?

Reply:
Temporary since 9.1 hasn't been tagged yet. Didn't see the other PR